index.html

```diff
@@ -4,51 +4,41 @@
     <meta charset="UTF-8" />
```

Comment:
- [ ] Please, could you consider the folder structure as it is required:
- you have to move `index.html`, `index.js`, and `index.css` into the `src` folder
<img width="168" alt="Screen Shot 2022-08-23 at 23 20 46" src="https://user-images.githubusercontent.com/49064106/186248179-e313e816-8c18-4b62-a837-008ea5df17ce.png">

- Please, consider this basic structure:
<img width="267" alt="Screen Shot 2022-08-23 at 23 24 54" src="https://user-images.githubusercontent.com/49064106/186248589-dc7ab053-d0f9-4da2-96a2-2fbe0102ad8e.png">